packages/workspace/src/tslint/nxEnforceModuleBoundariesRule.spec.ts

```diff
@@ -796,7 +796,7 @@ describe('Enforce Module Boundaries', () => {
       }
     );
     expect(failures[0].getFailure()).toEqual(
-      'Circular dependency between "mylibName" and "badcirclelibName" detected'
+      'Circular dependency between "mylibName" and "badcirclelibName" detected. The path is: mylibName -> badcirclelibName -> anotherlibName -> mylibName'
```

Comment:
How about one of these two messages:

Change the message to ... detected: myLibName -> badcirclarLib...

or

Change the message to ... detected. Path: myLibName -> badcirclarLib...

You don't normally put a colon after "is".